addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -242,12 +244,17 @@ private Devices loadDevices(Location location) throws GardenaException {
                     property.setAbility(ability);
 
                     // special conversion for pump, convert on/off to boolean
-                    if (device.getCategory().equals(DEVICE_CATEGORY_PUMP)
-                            && property.getName().equals(ABILITY_PUMP_ON_OFF)) {
-                        property.setValue(
-                                new PropertyValue(String.valueOf("on".equalsIgnoreCase(property.getValueAsString()))));
+                    if (device.getCategory().equals(DEVICE_CATEGORY_PUMP)) {
+                        if (property.getName().equals(PROPERTY_PUMP_ON_OFF)) {
+                            property.setValue(new PropertyValue(
```

Comment:
so what you actually do here is convert a string "on" to a string "true" can it work without this conversion?